src/AE2.lua

```diff
@@ -3,13 +3,20 @@ local ME = component.me_interface
 
 local AE2 = {}
 
-function AE2.requestItem(name, count)
+function AE2.requestItem(name, threshold, count)
     craftables = ME.getCraftables({
         ["label"] = name
     })
 
     if #craftables >= 1 then
         item = craftables[1].getItemStack()
+        itemInSystem = ME.getItemsInNetwork({
```

Comment:
Add a check whether the level even needs to be checked to not use unnecessary resources: `if threshold ~= nil then...`.

Reply:
Added in 685a8d5448f87ebc9c7a8b7816b175427b2d7476